libs/challenge-registry/user-profile/src/lib/_user-profile-theme.scss

```diff
@@ -8,64 +8,32 @@
   $warn: map.get($config, 'warn');
   $figma: map.get($config, 'figma');
 
-  .base-container1 {
-    border-color: transparent;
-    background-color: rgba(255, 255, 255, 1);
+  .base {
+    background-color: white;
   }
-
-  .base-top-section {
+  #top, #stats {
     border-color: transparent;
     background-color: map.get($figma, dl-color-default-hover1);
   }
-
-  .base-profile-pic {
-    border-color: transparent;
-    background-color: transparent;
-  }
-
-  .base-text {
-    color: rgba(0, 0, 0, 1);
-  }
-
-  .base-verified {
+  .verified {
     border-color: transparent;
     color: map.get($figma, dl-color-default-darkaccent3);
   }
-  .base-group {
-    border-color: transparent;
-  }
-  .base-text01 {
+  .username {
     color: map.get($figma, dl-color-default-secondary1);
   }
-  .base-usertypetag {
+  .user-tag {
     border-color: map.get($figma, dl-color-default-secondary2);
     background-color: map.get($figma, dl-color-default-accent4);
   }
-  .base-text02 {
-    color: rgba(0, 0, 0, 1);
-  }
-  .base-createdat {
-    border-color: transparent;
-  }
-  .base-text03 {
-    color: rgba(0, 0, 0, 1);
-  }
-  .base-text04 {
-    color: rgba(0, 0, 0, 1);
-  }
-  .base-updatedat {
+  .base-profile-pic {
     border-color: transparent;
-  }
-  .base-text05 {
-    color: rgba(0, 0, 0, 1);
-  }
-  .base-text06 {
-    color: rgba(0, 0, 0, 1);
+    background-color: transparent;
   }
   .base-basic-stats {
     background-color: map.get($figma, dl-color-default-hover2);
   }
-  .base-side-nav {
+  .sidenav {
```

Comment:
Since there is an extra `nav-group` selector added, change from `.sidenav` to `.nav-group`. So the sidebar's border and background colors match with what in figma.